dask_expr/_expr.py

```diff
@@ -2672,6 +2672,66 @@ def _fusion_pass(expr):
     return expr
 
 
+class FFill(MapOverlap):
+    _parameters = [
+        "frame",
+        "limit",
+    ]
+    _defaults = {"limit": None}
+    func = M.ffill
+    before = 1
+    after = 0
+    enforce_metadata = True
+
+    def _divisions(self):
+        return self.frame.divisions
+
+    @functools.cached_property
+    def _meta(self):
+        return self.frame._meta
+
+    def _simplify_up(self, parent):
+        if isinstance(parent, Projection):
+            return type(self)(self.frame[parent.operand("columns")], *self.operands[1:])
+
+    @functools.cached_property
+    def kwargs(self):
+        return dict(limit=self.limit)
+
+    def _lower(self):
+        return None
+
+    def _simplify_down(self):
+        self.before, self.after = 1 if self.limit is None else self.limit, 0
+
+        return MapOverlap(
+            frame=self.frame,
+            func=self.func,
+            before=self.before,
+            after=self.after,
+            meta=self._meta,
+            enforce_metadata=self.enforce_metadata,
+            kwargs=self.kwargs,
+        )
+
+
+class BFill(FFill):
+    func = M.bfill
+
+    def _simplify_down(self):
+        self.after, self.before = 1 if self.limit is None else self.limit, 0
```

Comment:
I meant making after and before actual properties that return the correct value based on limit

Reply:
https://github.com/dask-contrib/dask-expr/pull/454/commits/e8af6d6132c209c90f175703ce4126b98e25ff42